builds/jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -537,18 +585,18 @@ def setEventSourceMapping(eventSourceArn, config) {
 }
 
 def setLogStreamPermission(config){
-    if (config['service'] == "cloud-logs-streamer") {
-        def function_name = "${config_loader.INSTANCE_PREFIX}-${config['domain']}-${config['service']}-${current_environment}"
-        echo "set permission for cloud-logs-streamer"
-        try {
-            def rd = sh(script: "openssl rand -hex 4", returnStdout: true).trim()
-            sh "aws lambda add-permission --function-name arn:aws:lambda:${region}:${role_id}:function:${function_name} --statement-id lambdaFxnPermission${rd} --action lambda:* --principal logs.${region}.amazonaws.com --region ${region}"
-            echo "set permission for cloud-logs-streamer - success"
-        } catch (ss) {
-            //ignore if already registered permissions
-            echo(ss)
-            echo "set permission for cloud-logs-streamer - ok np"
-        }
-    }
+	if (config['service'] == "cloud-logs-streamer") {
```

Comment:
Revert the changes, I think tab was used to format this function while rest of the code used spaces. Let's use one convention (spaces) across the board.

Reply:
fixed